docs/sudoers_required_cmds.txt

```diff
@@ -0,0 +1,30 @@
+echo
+command
+grep
+/usr/sbin/dmidecode
+egrep
+ls
+yum
+tail
+/sbin/ifconfig
+sed
+locate
+find
+cat
+sort
+fgrep
+xargs
+dirname
+unzip
+java
+test
+/usr/bin/systemctl
+/sbin/chkconfig
+stat
+df
+ps
+tr
+readlink
+subscription-manager
+awk
+virt-what
```

Comment:
I know this file was manually created, but you may want to pass it through `| sort | uniq` before committing it to improve readability. Here's what I got:

```suggestion
/sbin/chkconfig
/sbin/ifconfig
/usr/bin/systemctl
/usr/sbin/dmidecode
awk
cat
command
df
dirname
echo
egrep
fgrep
find
grep
java
locate
ls
ps
readlink
sed
sort
stat
subscription-manager
tail
test
tr
unzip
virt-what
xargs
yum
```